CODEOWNERS

```diff
@@ -31,6 +31,26 @@
 /connectivity/check/ipcache.go @cilium/ipcache
 /connectivity/check/metrics*.go @cilium/metrics
 /connectivity/check/policy.go @cilium/sig-policy
+/connectivity/builder/** @cilium/ci-structure
+/connectivity/builder/all_ingress_deny_from_outside.go @cilium/sig-encryption
+/connectivity/builder/cluster_entity_multi_cluster.go @cilium/sig-clustermesh
+/connectivity/builder/dns_only.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_auth_always_fail.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_from_other_client_deny.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_from_outside.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_knp.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_l7.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_l7_named_port.go @cilium/sig-clustermesh
+/connectivity/builder/echo_ingress_mutual_auth_spiffe.go @cilium/sig-encryption
+/connectivity/builder/egress_gateway.go @cilium/sig-clustermesh
+/connectivity/builder/egress_gateway_excluded_cidrs.go @cilium/sig-clustermesh
+/connectivity/builder/health.go @cilium/sig-clustermesh
+/connectivity/builder/host_entity_egress.go @cilium/sig-clustermesh
+/connectivity/builder/host_entity_ingress.go @cilium/sig-clustermesh
```

Comment:
Not sure why these are assigned to @cilium/sig-clustermesh. The features they're testing don't seem related. Probably best to drop these entries and let @cilium/ci-structure take care of them for now.

Reply:
I just deleted them because `@cilium/ci-structure` is the default for the package.